### DISCUSSION TITLE
ScopeCollectionNotFound: Tried to access "mean" from collection "batch_stats" in "/BatchNorm_0" but the collection is empty.

### DISCUSSION THREAD

**Discussion by marcuswang6** [google/flax#2122] [Q&A] [answered]
Bug Code:
`res = nn.BatchNorm(use_running_average = True ,momentum = 0.1)(res)`

<img width="537" alt="image" src="https://user-images.githubusercontent.com/41324537/168425135-838e45f1-27fa-49f4-90e5-129ac64c1499.png">

Hi,
I face this error after adding a BatchNorm Layer and set use_running_average=True.
And I check the website where the bug leads me to, but still not know how to fix it.
It is a little confusing here, as I don't know what should be the mutable state.
Thanks in advance!

**Comment by marcuswang6** [comment, marked as answer]
preds, _ = Model().apply({'params': params}, input, mutable=['batch_stats'])
